<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>405</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="rButton">
      <property name="text">
       <string>remove letter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="sButton">
      <property name="text">
       <string>remove letter in place</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="kButton">
      <property name="text">
       <string>keep letter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pButton">
      <property name="text">
       <string>keep letter in place</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="letterLineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Begin</string>
   </property>
  </widget>
  <widget class="QListView" name="remainingWordsListView">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>241</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="displayRemWordsButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Show remaining words</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="placementSpinBox">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>111</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>270</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="numWordsRemainingLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>best word</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="bestWordLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>words remaining</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
